python/bigipconfigdriver.py

```diff
@@ -609,51 +845,60 @@ def _do_reset(self):
                     # Manually create custom profiles;CCCL doesn't yet do this
                     if 'customProfiles' in config['resources']:
                         for profile in config['resources']['customProfiles']:
-                            _create_client_ssl_profile(self._bigip, profile)
+                            _create_client_ssl_profile(
+                                self._cccls[0]._bigip_proxy.mgmt_root(),
+                                profile)
                             customProfiles = True
 
-                    cfg = create_config_kubernetes(self._bigip, config)
-                    if self._bigip.regenerate_config_f5(cfg):
+                    incomplete = 0
+                    for cccl in self._cccls:
+                        cfg = create_config_kubernetes(cccl, config)
+                        try:
+                            incomplete += cccl._apply_config(cfg)
+
+                            # Manually delete custom profiles (if needed)
+                            if customProfiles:
+                                _delete_client_ssl_profiles(
+                                    cccl, config['resources'])
+
+                        except F5CcclError as e:
+                            log.error("CCCL Error: %s", e.msg)
```

Comment:
This won't trigger backoff.  We have three error options in this program:

1. Error, trigger backoff and retry.
2. Error, log and do nothing (until next update).
3. Error, crash.

This is "class 2".  I think we should consider eliminating "class 2" and treat errors as "class 1" or "class 3".  What errors get us into this case and why should we not backoff or crash?